site/docs/community.md

```diff
@@ -0,0 +1,43 @@
+# Community
+
+Nessie is developed as a consensus-driven open source product under the Apache 2.0
+license. Development is done in the open leveraging GitHub issues, PRs and using
```

Comment:
```suggestion
license. Development is done in the open, leveraging GitHub issues, PRs and using
```
Not sure whether a comma's required here though

Reply:
don't think so. At least not in north american version of english

Reply:
TIL